src/client/src/tscGlobalmerge.c

```diff
@@ -605,8 +605,10 @@ static void doMergeResultImpl(SMultiwayMergeInfo* pInfo, SQLFunctionCtx *pCtx, i
     if (functionId < 0) {
       SUdfInfo* pUdfInfo = taosArrayGet(pInfo->udfInfo, -1 * functionId - 1);
       doInvokeUdf(pUdfInfo, &pCtx[j], 0, TSDB_UDF_FUNC_MERGE);
-    } else {
+    } else if (!TSDB_FUNC_IS_SCALAR(functionId)){
```

Comment:
The modification here is basically consistent with the original logic. Proposed change to:
`else {
  assert(!TSDB_FUNC_IS_SCALAR(functionId))
  aAggs[functionId].mergeFunc(&pCtx[j]);
}`

Reply:
yes. it will be modified